### PR TITLE
PageBrewingRecipe added 

### PR DESCRIPTION
**Pull request by Lothrazar**
This does not yet include a texture.  Using it will still pull in the furnace texture for now because of the arguments in Minecraft.getMinecraft().getTextureManager().bindTexture

 Wait to merge this until code reviewed and texture file updated,  Once texture is in i will have to revisit and adjust all the item positioning in the draw function

Note: I used the base class of BrewingRecipe instead of the IBrewingRecipe interface, because the interface does not have a getIngrediends method.  To compensate i also added a constructor that simply takes the three item stacks.

See comments in this issue https://github.com/TeamAmeriFrance/Guide-API/issues/65

### CODE REVIEW

**Comment by TehNut** [comment]
Just pulled the new textures in

**Comment by TehNut** [comment]
You can just call `SubTexture.POTION_GRID.draw(x, y)` to draw the proper texture, now.

**Comment by Lothrazar** [comment]
Ok ill pull from main 1.11 branch and update

**Comment by Lothrazar** [comment]
This is how it looks now.  I added healing potion recipe page to test book 2

@InsomniaKitten love your textures !

sample brewing

File: src/main/java/amerifrance/guideapi/page/PageBrewingRecipe.java [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+        GuiHelper.drawItemStack(input, x, y);
+        if (GuiHelper.isMouseBetween(mouseX, mouseY, x, y, 15, 15))
+          tooltip = GuiHelper.getTooltip(input);
+        x += hSpacing*2;
+        GuiHelper.drawItemStack(input, x, y);
+        if (GuiHelper.isMouseBetween(mouseX, mouseY, x, y, 15, 15))
+          tooltip = GuiHelper.getTooltip(input);
+        
+        
+        if (output == null || output.isEmpty())
```
Comment (TehNut) [inline src/main/java/amerifrance/guideapi/page/PageBrewingRecipe.java]: Remove this null check. I don't support people not fixing their things.

Comment (Lothrazar) [reply src/main/java/amerifrance/guideapi/page/PageBrewingRecipe.java]: righteo ... k updated

**Comment by Lothrazar** [comment]
Thanks for the merge. Ill start rolling with it once the maven updates <3

**Comment by Lothrazar** [comment]
Hey! So uhh, I don't want to be annoying, just wonder if you know when a new release & maven is coming out. 

 If its not for a while thats cool , i was just wondering if you know when.  👍  @TehNut

**Comment by TehNut** [comment]
To be honest, I legit just forgot to run a build

¯\\_(ツ)\_/¯

**Comment by TehNut** [comment]
Built

**Pull request merged**